lib/spack/env/cc

```diff
@@ -92,18 +95,22 @@ case "$command" in
     cc|c89|c99|gcc|clang|icc|pgcc|xlc)
```

Comment:
I fixed the compiler names to match latest develop again.  Note that they need to match the names in `lib/spack/env/*`, as each subdirectory there contains symlinks for different compiler names.